fe/src/components/base/search-bar.vue

```diff
@@ -1,8 +1,25 @@
 <template>
   <div class="flex space-x-2 p-2 border bg-white rounded-md">
     <Icon name="charm:search" class="text-disabled my-auto" />
-    <input placeholder="Search" class="bg-white outline-none" />
+    <input
+      v-model="searchKey"
+      placeholder="Search"
+      class="bg-white outline-none"
+      @keyup.enter="handleSumbit"
+    />
   </div>
 </template>
 
-<script setup></script>
+<script setup lang="ts">
+type Props = {
+  searchDefault: string;
```

Comment:
I think this should be optional because when user goes to `/campaigns` page, there's no search query param by default so it results to `Undefined` warning 🙇

![image](https://github.com/framgia/sph-solidity/assets/57890826/8429671b-9a3a-4226-843a-a25a1fdbd622)


```suggestion
  searchDefault?: string;
```

Reply:
Fixed in this [commit](https://github.com/framgia/sph-solidity/pull/48/commits/1e3d9e411c714ee4e639d04f0e2287189a6317eb)